coalib/parsing/DefaultArgParser.py

```diff
@@ -249,6 +249,15 @@ def default_arg_parser(formatter_class=None):
         ' additional arguments. If used without arguments, all bears specified'
         ' with --bears will be debugged (even implicit dependency bears).')
 
+    outputs_group.add_argument(
+        '--profile', nargs='?', const=True,
+        help='Enable bear profiling with cProfile, that can help to create'
+        ' a Performant Bears by profiling the Bear code. To specify where to'
```

Comment:
~~a Performant Bears~~ --> performant bears

Never use "a" for plurals. A plural form without "a" is equivalent and a grammatical correct sentence.